lisa/tools/ethtool.py

```diff
@@ -54,19 +53,19 @@ def __init__(self, interface: str, device_channel_raw: str) -> None:
         self._parse_channel_info(interface, device_channel_raw)
 
     def _parse_channel_info(self, interface: str, raw_str: str) -> None:
-        current_settings = _current_settings_pattern.match(raw_str)
-        max_settings = _max_settings_pattern.match(raw_str)
+        current_settings = re.search(_current_settings_pattern, raw_str)
```

Comment:
1. What's the reason to change match to search? It looks similar here.
2. It can be called like _current_settings_pattern.search(raw_str). no need to use static method.

Reply:
> 1. What's the reason to change match to search? It looks similar here.
      _current_settings_pattern.match() wasn't matching (may be because it was using DOTALL flag) not sure but it was giving error.

> 2. It can be called like _current_settings_pattern.search(raw_str). no need to use static method.
ok will use it like this, test and update PR.